Dockerfile.opendistroforelasticsearch

```diff
@@ -0,0 +1,10 @@
+FROM amazon/opendistro-for-elasticsearch:1.6.0
+
+WORKDIR /usr/share/elasticsearch/
+
+COPY config/opendistroforelasticsearch /usr/share/elasticsearch/config
+COPY bin/opendistroforelasticsearch/create-index.sh /usr/share/elasticsearch/create-index.sh
+
+ENV http.port=19200
+ENV opendistro_security.disabled=true
+ENV knn.algo_param.index_thread_qty=4
```

Comment:
Does this only control indexing or also searches? I would rather see this set to 1, especially if it's multi-threaded query execution.

Reply:
I think that [knn.algo_param.index_thread_qty](https://opendistro.github.io/for-elasticsearch-docs/docs/knn/settings/) parameter is only for indexing.

Ref: https://github.com/nmslib/nmslib/blob/master/manual/latex/methods.tex#L1198-L1205

If we would like to improve write performance then we increase it.